hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieCommitMetadata.java

```diff
@@ -42,7 +43,9 @@
     private static volatile Logger log = LogManager.getLogger(HoodieCommitMetadata.class);
     protected HashMap<String, List<HoodieWriteStat>> partitionToWriteStats;
 
-    private HashMap<String, String> extraMetadataMap;
+    protected HashMap<String, String> extraMetadataMap;
+
+    protected String currentBasePath;
```

Comment:
Its okay to not have basePath here, since we know it anyways from HoodieWriteClient or HoodieReadClient input configs